libmscore/synthesizerstate.cpp

```diff
@@ -54,7 +55,15 @@ void SynthesizerState::read(XmlReader& e)
                   else
                         e.unknown();
                   }
-            push_back(group);
+            tempGroups.push_back(group);
+            }
+
+      if (int(tempGroups.size()) > 0) {
```

Comment:
Why not just `if (!tempGroups.empty())`?

Reply:
same difference :)

I can change it if you really want....

Reply:
Well, at least it would allow to avoid that `int` conversion. I would also consider the option with `empty()` somewhat more explicit and thus more clear but that is probably a matter of preference.